Content.Tests/Shared/Utility/ContentHelpers_Test.cs

```diff
@@ -82,6 +82,58 @@ public class ContentHelpers_Test
                 (320, 6, 5, 5),
             };
 
+        public static readonly IEnumerable<(double val, double max, int levels, int expected)> TestEqLvls = 
+            new (double, double, int, int)[]
+            {
+                // Testing odd max on even levels
+                (-10, 5, 2, 0),
+                (0, 5, 2, 0),
+                (1, 5, 2, 0),
+                (2, 5, 2, 0),
+                (3, 5, 2, 1),
+                (4, 5, 2, 1),
+                (5, 5, 2, 1),
+ 
+                // Testing even max on odd levels
+                (0, 6, 3, 0),
+                (1, 6, 3, 0),
+                (2, 6, 3, 1),
+                (3, 6, 3, 1),
+                (4, 6, 3, 2),
+                (5, 6, 3, 2),
+                (6, 6, 3, 2),
+                
+                // Testing even max on even levels
+                (0, 4, 2, 0),
+                (1, 4, 2, 0),
+                (2, 4, 2, 1),
+                (3, 4, 2, 1),
+                (4, 4, 2, 1),
+                
+                // Testing odd max on odd levels
+                (0, 5, 3, 0),
+                (1, 5, 3, 0),
+                (2, 5, 3, 1),
+                (3, 5, 3, 1),
+                (4, 5, 3, 2),
+                // Largers odd max on odd levels
+                (0, 7, 3, 0),
+                (1, 7, 3, 0),
+                (2, 7, 3, 0),
+                (3, 7, 3, 1),
+                (4, 7, 3, 1),
+                (5, 7, 3, 2),
+                (6, 7, 3, 2),
+                (7, 7, 3, 2),
+                
+                // Testing edge cases
+                (0.1, 6, 5, 0),
+                (-32, 6, 5, 0),
+                (2.4, 6, 5, 1),
+                (2.5, 6, 5, 2),
+                (320, 6, 5, 4),
+            };
+        
```

Comment:
This can probably be replaced with [TestCase(-10, 5, 2, 0)] placed above TestEqual in place of [Test]